ui/src/elements/ngm-object-information.ts

```diff
@@ -44,13 +45,22 @@ export class NgmObjectInformation extends LitElementI18n {
 
   render() {
     this.hidden = !this.opened;
-    if (this.info && (this.info.popupContent || this.info.properties)) {
-      const content = this.info.popupContent ?
-        unsafeHTML(this.info.popupContent) :
+    if (this.info && (this.info.popupItems?.length || this.info.properties)) {
+      const content = this.info.popupItems?.length ?
+        this.info.popupItems.map(it => html`
+          <div class="query-list-item" @mouseenter=${() => it.mouseEnter()} @mouseleave=${() => it.mouseLeave()}>
+            <div class="item-zoom-btn-container" ?hidden="${!it.zoom}">
+              <button @click="${it.zoom}" class="ui button ngm-zoom-obj-btn ngm-action-btn">
+                ${i18next.t('obj_info_zoom_to_object_btn_label')}
+                <div class="ngm-zoom-plus-icon"></div>
+              </button>
+            </div>
+            ${unsafeHTML(it.content)}
+          </div>`) :
         html`
           <table class="ui compact small very basic table">
             <tbody>
-            ${this.info.properties.map(row => {
+            ${this.info.properties?.map(row => {
```

Comment:
If `this.info.properties` is `undefined`, it will display "undefined" in the rendered HTML, right?